src/Stratis.Sidechains.Networks/FederatedPegNetwork.cs

```diff
@@ -56,7 +54,7 @@ public static Block CreateGenesis(SmartContractPoAConsensusFactory consensusFact
             genesis.Header.HashPrevBlock = uint256.Zero;
             genesis.UpdateMerkleRoot();
 
-            ((SmartContractBlockHeader)genesis.Header).HashStateRoot = new uint256("21B463E3B52F6201C0AD6C991BE0485B6EF8C092E64583FFA655CC1B171FE856");
+            ((SmartContractPoABlockHeader)genesis.Header).HashStateRoot = new uint256("21B463E3B52F6201C0AD6C991BE0485B6EF8C092E64583FFA655CC1B171FE856");
```

Comment:
I see this is unchanged, wondering if this is OK ?

Reply:
yeah I think that's the default thing.